src/components/PriceRangePlot/PriceRangePlot.tsx

```diff
@@ -278,7 +283,7 @@ export const PriceRangePlot: React.FC<IPriceRangePlot> = ({
         ]}
         curve='basis'
         margin={{ top: isSmDown ? 55 : 25, bottom: 15 }}
-        colors={[colors.invariant.accent1, colors.invariant.accent2, colors.invariant.accent1]}
+        colors={[colors.invariant.green, colors.invariant.green, colors.invariant.green]}
```

Comment:
color for elements 0 and 2 should be pink

Reply:
done